notebooks/255-mms-massively-multilingual-speech/255-mms-massively-multilingual-speech.ipynb

```diff
@@ -0,0 +1,676 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.        if not asr_model_xml_path.exists():
</pre><p>Does asr_model.load_adapter need only for conversion?</p><p> </p><p>If yes, then probably we can move adapter loading under condition that model is not converted yet</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1257/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
Moved